Makefile

```diff
@@ -183,6 +185,22 @@ ci-rustdoc:
 audit:
 	@for f in `./tools/list_lock.sh`; do echo "$$(tput bold)Auditing $$f"; (cd "$$f" && cargo audit || exit 1); done
 
+.PHONY: emulation-setup
+emulation-setup:
+	@#Use the latest QEMU as it has OpenTitan support
+	@if [[ ! -d qemu || ! -f qemu/VERSION ]]; then \
+		rm -rf qemu; \
+		git clone https://github.com/qemu/qemu.git; \
```

Comment:
probably better to clone this into `tools/qemu` rather than adding stuff to top-level directory; we install a bunch of other stuff into `tools` already